Lib/fontTools/cu2qu/__main__.py

```diff
@@ -1,6 +1,7 @@
+"""Convert a UFO font with cubic curves to quadratic curves"""
+
 import sys
 from .cli import main
 
-
```

Comment:
I’m sure @anthrotype will tell you that there should be two spaces after import statements based on some PEPsomething 😇

Reply:
One more time then...